javalib/src/main/scala/java/util/UUID.scala

```diff
@@ -134,14 +135,35 @@ object UUID {
   private final val NameBased = 3
   private final val Random = 4
 
-  private lazy val rng = new Random() // TODO Use java.security.SecureRandom
+  private lazy val rng = new SecureRandom()
 
   def randomUUID(): UUID = {
```

Comment:
Even though we do not normally document out Java library implementation, they do get published. I recommend adding a really good java/scaladoc outlining that this uses `java.security.SecureRandom` as a source of random numbers and that a third party library is required to use this method.